### PR TITLE
added usage of subtest for etl

### PR DESCRIPTION
**Pull request by eklatzer**
One of #2098

### CODE REVIEW

**Comment by eklatzer** [comment]
CI error: `--- FAIL: TestSeedWithTime (0.00s)  animal_magic_test.go:22: SeedWithTime always sets the same seed`
The example uses `time.Now().UnixNano())` so it should be fine, therefore I added a constant offset and the test should not fail again

File: exercises/concept/animal-magic/animal_magic_test.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ func TestSeedWithTime(t *testing.T) {
 			return
 		}
 		last = got
-		time.Sleep(time.Duration(rand.Intn(10)) * time.Microsecond)
+		time.Sleep((time.Duration(rand.Intn(10)) + 10) * time.Microsecond)
```
Comment (andrerfcsantos) [inline exercises/concept/animal-magic/animal_magic_test.go:20]: I think this file was committed by mistake?

File: exercises/practice/etl/.meta/config.json [hunk 2]
```diff
@@ -33,4 +34,4 @@
   },
   "source": "The Jumpstart Lab team",
   "source_url": "http://jumpstartlab.com"
-}
+}
```
Comment (andrerfcsantos) [inline exercises/practice/etl/.meta/config.json:37]:
```suggestion
}

```

**Review by andrerfcsantos — COMMENTED** [review]
Just an extra file here and a file missing a newline at the end.

**Review by andrerfcsantos — APPROVED** [review]

**Pull request merged**